packages/@webex/internal-plugin-dss/test/unit/spec/dss.ts

```diff
@@ -239,6 +246,61 @@ describe('plugin-dss', () => {
 
         expect(result).to.be.null;
       });
+
+      it('fails with default timeout when mercury does not respond and the timeout option is not set', async () => {
+        const {promise} = await testMakeRequest({
+          method: 'lookupDetail',
+          resource: '/lookup/orgid/userOrgId/identity/test id/detail',
+          params: {id: 'test id'},
+          bodyParams: {},
+        });
+
+        clock.tick(webex.config.dss.requestTimeout);
```

Comment:
I'd prefer explicit time (1000) rather than assuming the config is set correctly.